frontend/src/src/components/LandingPage/index.js

```diff
@@ -1,106 +1,149 @@
-import React, { useState, useEffect } from 'react';
-import { Redirect } from 'react-router-dom';
-import axios from 'axios';
-import { Button, Modal } from 'react-bootstrap';
-import Form from 'react-bootstrap/Form';
-import { useAuth } from "../../context/auth";
+import React, {useCallback, useEffect, useState} from 'react';
+import {useHistory} from 'react-router-dom';
+import {Button, Form, Modal} from 'react-bootstrap';
+import {useAuth} from "../../context/auth";
+import api from "../../services/api";
+import {useIsMounted} from '../../hooks/useIsMounted'
+import {useUser} from "../../context/user";
+
+// TODO: If we add the ability to select an organization, that would happen in this component.
 
 export default function LandingPage() {
     const [attorneyData, setAttorneyData] = useState([]);
-    const [attorneyKey, setAttorneyKey] = useState(0);
-    const [isAttorneyChosen, setAttorneyChosen] = useState(false);
-    const [profileGenerated, setProfileGenerated] = useState(false);
+    const [attorneyKey, setAttorneyKey] = useState(""); // PK of the selected attorney
     const [isError, setIsError] = useState(false);
-    const { authTokens } = useAuth();
+    const [errorMessage, setErrorMessage] = useState("");
+
+    const {isAuthenticated, authenticatedRequest} = useAuth();
+    const {refreshUserProfile} = useUser();
+    const history = useHistory();
+    const getIsMounted = useIsMounted();
 
-    // useEffect is the React Hook equivalent to ComponentDidMount
+    // Fetches the list of attorneys to populate form options.
     useEffect(() => {
-        const token = `Bearer ${authTokens.access}`;
-        var config = {
-            'headers': { 'Authorization': token }
-        };
+        if (isAuthenticated) {
```

Comment:
This block really shows where getIsMounted is killing you--you have that check in 3 places in this function. A good activity for a future ticket is removing one of them, recreating the warning, and then seeing what's causing us to unmount things before we operate on them